test/strongbox/gui_test.clj

```diff
@@ -44,3 +44,27 @@
     (let [cases [[:foo :#foo]]]
       (doseq [[given expected] cases]
         (is (= expected (gui/as-selector given)))))))
+
+(deftest gui-select-installed-addons
+  (testing "selecting installed addons using the gui selects the corresponding installed addons"
```

Comment:
"selecting addons using the gui selects the corresponding installed addons in the application state"